### PR TITLE
Calculator - Lunch level

### PR DESCRIPTION
**Pull request by camarleigh**
HI there. This is calculator, lunch level.

### CODE REVIEW

File: calclun.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# modifications from breakfast program: 
+# 1) replaces statement "Adding these numbers..." with actual equation.
+# 2) moves greeting outside of 'while...end' condition loop so it will not be 
+#    repeated over and over again when/if user decides to try program multiple times. 
+
+
+ puts "Welcome to the BEST CALCULATOR EVER!"
+
+answer = "y"
+while answer == "y"
```
Comment (ahsterling) [inline calclun.rb:51]: Your while loop here is a nice way to allow the user to continue using the calculator for as long as they want - good job!

File: calclun.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+# Lunch version
+# modifications from breakfast program: 
+# 1) replaces statement "Adding these numbers..." with actual equation.
+# 2) moves greeting outside of 'while...end' condition loop so it will not be 
+#    repeated over and over again when/if user decides to try program multiple times. 
+
+
+ puts "Welcome to the BEST CALCULATOR EVER!"
+
```
Comment (ahsterling) [inline calclun.rb:49]: you could also call the `answer` variable something like `continue` to clarify that it determines whether or not the user wants to continue playing the game

File: calclun.rb [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+# "if the operation equals add OR the operation equals +, then....adds ...., etc."
+
+# question: could a hash be used to handle the entry of almost any version of
+# how the operation is called? Thus eliminating repitition of current if..end condition? 
+# e.g.  if operation == {add, addition, plus, +, subtract, times, *, multiply, etc. } 
+# answer: no. Not quite sure why. tried to replace and suggestions with both hash and array,
+# no luck :( Will revisit at a later date.  [error msg: unexpected ","]
+
+  if operation == "add"|| operation == "+"
+    then puts "#{num_one} + #{num_two} = #{num_one + num_two}"
```
Comment (ahsterling) [inline calclun.rb]: I don't think the `then

Comment (ahsterling) [inline calclun.rb:76]: I don't think the `then` keyword here is strictly necessary, you could simply write this:

```
if operation == "add" || operation == "+"
    puts "#{num_one} + #{num_two} = #{num_one + num_two}"
```

**Comment by ahsterling** [comment]
Awesome job!  You did a great job making your code readable with good variable names and indentation, and your use of the while loop is a good way to increase the usability.  You can see some small suggestions in the comments above, but overall this was great!

**Comment by camarleigh** [comment]
Hi Allie,

thanks for the review! Yeah I noticed when I was pushing it, the "thens", and I was like " where did those come from?" I'm pretty sure we never covered then so much as me saying it (what I thought was) silently in my head. I also like the idea of naming the variable "answer" continue. I will change that but no need to re- review.

Thanks again, m

**Comment by ahsterling** [comment]
No problem :)  Great job!

**Pull request merged**